corrected typo in send_for_identification function

ashfaq92 opened the pull request:
changed send_for_identificattion  to send_for_identification

ashfaq92 committed:
corrected typo in send_for_identification function

changed send_for_identificattion  to send_for_identification

xorwen commented:
Thanks @ashfaq92 :). Could you also change the function call (L61) and add it into your push request?

ashfaq92 committed:
corrected typos (2)

converted send_for_identificattion to send_for_identification on L#35 and L#61 too

ashfaq92 committed:
Update plant_id_client.py
